src/main/suitable/compliment/sources/cljs.clj

```diff
@@ -2,10 +2,10 @@
   "Standalone auto-complete library based on cljs analyzer state"
   (:refer-clojure :exclude [meta])
   (:require [clojure.set :as set]
-            #?(:clj [compliment.sources :refer [defsource]])
```

Comment:
I changed a couple files from .cljc to .clj because the cljs branches were visibly incomplete... I guess it was drafted code

It seems better to leave things less confusing for maintainers - we shouldn't maintain code that doesn't run :)